misc/identity-keys/src/identity-keys.ts

```diff
@@ -66,15 +66,16 @@ export class IdentityKeys implements IIdentityKeys {
       try {
         const [pubKeyHex, privKeyHex] = await this.generateIdentityKey();
         const didKey = encodeEd25519Key(pubKeyHex);
+        const domain = new URL(this.keyserverUrl).host;
```

Comment:
Since working with `URL` already, would be nice to add the port number here if non-default like I did over [here](https://github.com/WalletConnect/notify-server/pull/65/files#diff-eb771f54a1fdc1c330a82981d7d27d1202a5cbe3a59581ff67af5e3361f5f90eR180-R187).

Reply:
`URL().host` includes the port number if non standard: https://developer.mozilla.org/en-US/docs/Web/API/URL

Hence chose `host` over `hostname` (which does not include port)